packages/checkout-storefront/src/components/SelectBox/SelectBox.tsx

```diff
@@ -1,34 +1,32 @@
 import clsx from "clsx";
 import { Children, Classes } from "@/checkout-storefront/lib/globalTypes";
-import { FC } from "react";
+import { FC, HTMLAttributes } from "react";
 
-export interface SelectBoxProps extends Classes, Children {
+export interface SelectBoxProps
+  extends Classes,
+    Children,
+    Omit<HTMLAttributes<HTMLInputElement>, "children"> {
```

Comment:
I think this one is more clear, because DOM types look exactly the same (`HTML...`). If we'd use `React.` namespace, it would be more clear that the types came from React
```suggestion
    Omit<HTMLAttributes<React.HTMLInputElement>, "children"> {
```